start-site/src/main/resources/application.yml

```diff
@@ -143,7 +143,7 @@ initializr:
         versionProperty: timefold-solver.version
         mappings:
           - compatibilityRange: "[3.2.0,3.4.0-M1)"
-            version: 1.13.0
+            version: 1.14.0
```

Comment:
[I can't find 1.14.0 for timefold solver](https://central.sonatype.com/artifact/ai.timefold.solver/timefold-solver-bom/1.13.0). Where did you find that?

Reply:
My mistake. I though I found it in [https://mvnrepository.com/](https://mvnrepository.com/) but the latest version is 1.13.0...
Sorry about that